src/components/filter/filter.css

```diff
@@ -0,0 +1,72 @@
+.filter-container {
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+  position: fixed;
+  left: 60%;
+  bottom: 60%;
+  display: none;
+  flex-direction: column;
```

Comment:
I guess you do not need `flex-direction` property if `display `is set to `none`

Reply:
Yes, but it would be needed when user will click on filter button. Coz in that case display will be flex.